src/client/components/AlvisGraph.tsx

```diff
@@ -416,12 +420,29 @@ export class AlvisGraph extends React.Component<
     nextContext: any,
   ) {
     console.log('ATTENTION!!! shouldComponentUpdate');
-    return false;
+    return true;
   }
 
   // componentWillUpdate?(nextProps: P, nextState: S, nextContext: any): void;
-  // componentDidUpdate?(prevProps: P, prevState: S, prevContext: any): void;
-  // componentWillUnmount?(): void;
+  componentDidUpdate(prevProps) {
+    const { active } = this.props;
+
+    this.graph.setEnabled(this.props.active);
+
+    if (active && !prevProps.active) {
+      this.outline.refresh();
```

Comment:
It seems that during switching between tabs (graphs) `react-bootstrap` sets `visible: hidden` or something similar too deep in svg and on turning back it does not unset all of them thus we force refresh.